### DISCUSSION TITLE
Docs accessible in other panel (bottom or side)!

### DISCUSSION THREAD

**Discussion by LokyStaz** [godotengine/godot-proposals#7370] [GUI]
Hi,
I think it would be nice to be able to open the doc in the bottom panel instead of the script panel...
Like this we can see it in same time as we see our code!
It's so frustrating that I always have to switch between the code and the doc, it kind of go against the point to have the doc direct inside godot..

for example, when working on laptop (one small screen), it's faster to switch to the doc in the browser (alt tab) than to use the mouse to clic between the script and the doc. And both technique hide the code for the time reading the doc anyway.

So yeah to me the point to have the doc inside godot would be to see the code and the doc in same time, it would greatly improve the workflow by no switching between code and doc !!

**Comment by Calinou** [comment]
In Godot 4.1 and later, you can split the script editor to a floating window.

The bottom panel is pretty small in terms of height if you don't maximize it, so I'm not sure if it would work well as a documentation reader (especially on a laptop screen).

**Reply by Zireael07** [reply]
Seconded, and a maximized bottom panel covers the script panel so you can't see both at the same time

**Comment by LokyStaz** [comment]
So... Perhaps a side panel, like a tab on the inspector dock... 
For people who respect line length of maxi 100, to expand the inspector dock (or other side dock) would make the docs readable !!

Something like that: [screenshot](https://i.imgur.com/HE5tX80.png)

**Reply by Calinou** [reply]
> So we can't edit discussion... Art!! I have to more careful so!

You can edit messages using the **…** icon in the top-right corner of each message.

**Reply by LokyStaz** [reply]
thanks!  I didn't saw it this morning ^^'

**Reply by Calinou** [reply]
Regarding your request, it could be done if documentation is available in the bottom panel and https://github.com/godotengine/godot/pull/36040 is merged. That said, the way the editor help is implemented would have to be factored out to prevent code duplication. It's a lot of effort for relatively limited benefit, as we should *not* remove the ability to read documentation in the Script tab (a lot of people are used to this workflow).
